test/fixture/fixtures/test.js

```diff
@@ -0,0 +1,6 @@
+import test from '../../';
```

Comment:
I think the fixtures should be placed in a super folder with a descriptive name. Like `test/fixture/ignored-dirs/fixtures/test.js`. Right now it looks like it's for something else. Same with the below.